src/type/scalars.js

```diff
@@ -121,3 +144,45 @@ export const GraphQLID = new GraphQLScalarType({
       null;
   }
 });
+
+export const GraphQLDateTime = new GraphQLScalarType({
+  name: 'DateTime',
+  description: 'An ISO-8601 encoded UTC date string.',
+  serialize(value: mixed): string {
+    if (!(value instanceof Date)) {
```

Comment:
I think `serialize` should be more accepting of input than just asserting `Date` instances. For example, iso date strings should probably also be supported here.

Also, I'm curious how convenient we should try to be for unix timestamps? It's *really* common for datetimes to actually be stored as unix times in databases, and I fear that if we don't make it easy to represent these as `DateTime` that they'll be lazily be represented as `Int` instead. What do you think?